arch/arm/core/Kconfig

```diff
@@ -29,6 +29,98 @@ config CPU_CORTEX_M
 	help
 	  This option signifies the use of a CPU of the Cortex-M family.
 
+config CPU_CORTEX_R
+	bool
+	select CPU_CORTEX
+	select HAS_FLASH_LOAD_OFFSET
+	select ARCH_HAS_STACK_PROTECTION if ARM_MPU
+	select ARCH_HAS_USERSPACE if ARM_MPU
```

Comment:
Here, you imply that ARM_MPU may be selected for Cortex-R, but in CMakeLists, CONFIG_ARM_MPU brings-in cortex_m/mpu folder

Reply:
I removed these two lines.  Just a heads-up that we are working on MPU support for Cortex-R so that code will be submitted at some point.